.github/workflows/tests.yml

```
@@ -48,6 +48,36 @@ jobs:
         # TODO: move this to its own action
         npm install @npmcli/arborist@7 semver@7 --no-save
         node .github/scripts/check-engines.js
+  Pack_Test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+    - name: Use Node.js 20.x
+      uses: actions/setup-node@v3
```

This update should happen in most of our GHAs.
```suggestion
      uses: actions/setup-node@v4
```
https://github.com/actions/setup-node/releases

---

Fixed